frontend/src/pages/details-page.tsx

```diff
@@ -17,12 +17,12 @@ export default function DetailsPage() {
     const getPostById = async () => {
       try {
         await axios.get(import.meta.env.VITE_API_PATH + `/api/posts/${postId}`).then((response) => {
-          console.log(response.data);
+          // console.log(response.data);
           setIsLoading(false);
           setPost(response.data);
         });
       } catch (error) {
-        console.log(error);
+        // console.log(error);
       }
```

Comment:
same

Reply:
This also was suggested by lighthouse, and It's generally not recommended to keep console.log in production because it can confuse users and potentially create security risks.